style.css

```diff
@@ -503,3 +503,91 @@ body {
   padding-top: 1px;
   margin-left: 5px;
 }
+
+.contact-form {
+  margin: 0 24px 0 18px;
+  font-family: Poppins, sans-serif;
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-box {
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  height: 779px;
+}
+
+.contact-section {
+  background-color: #fff;
+  background-image: url("assets/images/contact-form-shapes-mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-position: top right;
+  width: 380px;
+  height: 779px;
+}
```

Comment:
- [ ] Kindly add the background images to the contact section to match the Figma design.
- Your design:
<img width="167" alt="image" src="https://user-images.githubusercontent.com/77190203/155216729-71a70c55-d60e-4e94-89bf-9e59f875613e.png">

Reply:
Addressed 💪